src/lib/apiFetchStreamableSong.js

```diff
@@ -0,0 +1,6 @@
+import Promise from 'bluebird';
+import { getWithAuth } from './apiService';
+
+export const apiFetchStreamableSongs = (token, path) => new Promise((resolve, reject) => {
+  getWithAuth(`/api/file${path}`, token, resolve, reject);
```

Comment:
Abstract this path into a constant :)

Reply:
do you mean like a variable?
**const routeWithPath = /api/file${path};**

or a constant in routes.js?
**export const API_FILE_ROUTE = `/api/file`**

Reply:
a constant in routes.js :)